### ISSUE TITLE
Search results for Places should include Geometry (coordinates)

### ISSUE THREAD

**Issue by Libisch** [Beit-Hatfutsot/dbs-back#139]
**Expected:**
Places response should include geo data ("Geometry") in both search results and item pages.

**Actual:**
A significant portion of (Places) items in search results pages don't return geo data. The geo data exists for the same items in item  pages.

**Comment by Libisch** [comment]
@TheGrandVizier

**Comment by OriHoch** [comment]
@Libisch please add some specific examples - with links to the relevant API calls where the problem can be seen

**Comment by Libisch** [comment]
**Example:**
"Akmene" item page result:
``` json
[
	{
		"LocationInMuseum": null,
		"Pictures": [],
		"PictureUnitsIds": null,
		"UpdateDate": "2015-08-13 11:42:00",
		"OldUnitId": "HB068227.HTM-EB002500.HTM",
		"id": 191895,
		"UpdateUser": "simona",
		"PrevPictureFileNames": null,
		"PrevPicturePaths": null,
		"PlaceTypeCode": 2,
		"TS": "00000000004039b9",
		"PlaceTypeDesc": {
			"En": "Town",
			"He": "עיירה"
		},
		"UnitType": 5,
		"UnitTypeDesc": "Place",
		"EditorRemarks": "hasavot from Places ",
		"DisplayStatusDesc": "Museum only",
		"RightsDesc": "Full",
		"Bibiliography": {
			"En": "YAHADUT LITA BOOKS",
			"He": "ספרי יהדות ליטא"
		},
		"UnitText1": {
			"En": "AKMENE(...)",
			"He": "אוקמיא(..)."
		},
		"UnitText2": {
			"En": null,
			"He": null
		},
		"UnitPlaces": [],
		"UnitStatus": 3,
		"UnitHeaderDMSoundex": {
			"En": "ZZ E056600 ZZ ",
			"He": "ZZ H056160 H056600 ZZ "
		},
		"UnitDisplayStatus": 2,
		"RightsCode": 1,
		"UnitId": 73462,
		"IsValueUnit": true,
		"StatusDesc": "Completed",
		"PlaceParentTypeCodeDesc": {
			"En": "Country"
		},
		"PlaceParentId": 70518,
		"UserLexicon": null,
		"Attachments": [],
		"geometry": {
			"type": "Point",
			"coordinates": [
				22.7545972,
				56.2460515
			]
		},
		"Slug": {
			"En": "place_akmene",
			"He": "מקום_אוקמיאן"
		},
		"Header": {
			"En": "AKMENE",
			"He": "אוקמיאן"
		},
		"ForPreview": false,
		"_id": "73462"
	}
]
```
"Akmene" From search results:
``` json
{
				"_score": 27.11805,
				"_type": "places",
				"_id": "73462",
				"_source": {
					"LocationInMuseum": null,
					"Pictures": [],
					"PictureUnitsIds": null,
					"UpdateDate": "2015-08-13T11:42:00",
					"OldUnitId": "HB068227.HTM-EB002500.HTM",
					"dm_soundex": [
						"056600"
					],
					"id": 191895,
					"UpdateUser": "simona",
					"PrevPictureFileNames": null,
					"PrevPicturePaths": null,
					"PlaceTypeCode": 2,
					"TS": "00000000004039b9",
					"PlaceTypeDesc": {
						"En": "Town",
						"He": "עיירה"
					},
					"UnitType": 5,
					"UnitTypeDesc": "Place",
					"EditorRemarks": "hasavot from Places ",
					"UnitStatus": 3,
					"RightsDesc": "Full",
					"Bibiliography": {
						"En": "YAHADUT LITA BOOKS",
						"He": "ספרי יהדות ליטא"
					},
					"UnitText1": {
						"En": "AKMENE(...)",
						"He": "אוקמיאן(...)"
					},
					"UnitText2": {
						"En": null,
						"He": null
					},
					"UnitPlaces": [],
					"DisplayStatusDesc": "Museum only",
					"UnitDisplayStatus": 2,
					"RightsCode": 1,
					"UnitId": 73462,
					"IsValueUnit": true,
					"StatusDesc": "Completed",
					"PlaceParentTypeCodeDesc": {
						"En": "Country"
					},
					"PlaceParentId": 70518,
					"UserLexicon": null,
					"Attachments": [],
					"Slug": {
						"En": "place_akmene",
						"He": "מקום_אוקמיאן"
					},
					"Header": {
						"En": "AKMENE",
						"He": "אוקמיאן"
					},
					"ForPreview": false
				},
				"_index": "mojp-live"
			}
```

**Comment by OriHoch** [comment]
thanks @Libisch  - opened #141 to fix this case of mongo having the item but elastic doesn't

need to see if there are more cases

**Comment by OriHoch** [comment]
assuming there are no more problems, this issue was fixed in #141 
it's deployed to dev (release 0.12.4)
